promscale/scale-ha/multi-node.md

```diff
@@ -22,3 +22,5 @@ SELECT add_prom_node('example_node_name');
 Make sure that you run the `add_prom_node` command as the same database user as
 the one writing data from Promscale.
 </highlight>
+
+[db-multinode]: https://docs.timescale.com/latest/getting-started/setup-multi-node-basic
```

Comment:
@Loquacity should this url be an absolute reference to the prod documentation or use a relative path?